e2e/pages/app/number-picker/readonly/number-picker-read-only.testpage.component.html

```diff
@@ -0,0 +1,16 @@
+<a id="top-focus" href="javascript:void(0)">Top focus target</a>
+
+<div id="root">
+
+    <form [formGroup]="form">
+        <p>readonly:</p>
+        <ux-number-picker id="read-only-number-picker" class="number-picker-component-demo" min="0" max="10" [readonly]="readonly" formControlName="readonly"></ux-number-picker>
```

Comment:
Since you are visually testing, you should also include `ux-number-picker-inline`. It has a separate appearance in the Micro Focus theme.

I also notice that you have set min and max both in the markup and in the form builder, with conflicting values. This test does not cover min/max so you can remove all of that. Best to keep tests simple.